aio/aio-proxy/aio_proxy/doc/open-api.yml

```diff
@@ -264,6 +264,15 @@ paths:
           example: "0022004T"
           schema:
             type: string
+        - name: nature_juridique
+          in: query
+          description: >-
```

Comment:
Maybe a link to a valid list ?

Reply:
I couldn't find a full list of values. https://www.insee.fr/fr/information/2028129